### PR TITLE
add com.github.artemanufrij.graphui

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Add https://github.com/artemanufrij/graphui by @artemanufrij

The app uses `--filesystem=home` until I patch it upstream to use `Gtk.NativeFileChooser`

### CODE REVIEW

File: com.github.artemanufrij.graphui.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+modules:
```
Comment (TingPing) [inline com.github.artemanufrij.graphui.yml]: Missing `cleanup`.

**Comment by nedrichards** [comment]
bot, build com.github.artemanufrij.graphui

**Comment by bilelmoussaoui** [comment]
Builds perfectly. Going to take a look at the cleanup right now.

**Comment by barthalion** [comment]
Any news?

**Comment by bilelmoussaoui** [comment]
Totally forgot about it. Will fix it today.

**Comment by barthalion** [comment]
@bilelmoussaoui Gentle reminder :stuck_out_tongue:

**Comment by bilelmoussaoui** [comment]
ha! I always see this PR when I check new ones and I tell to myself, will do that in a few minutes and totally forgot about it. Will fix that right now!

**Comment by bilelmoussaoui** [comment]
testbot, build com.github.artemanufrij.graphui

**Comment by nedrichards** [comment]
bot, build com.github.artemanufrij.graphui

**Comment by nedrichards** [comment]
I can't get this to actually render an example and get errors like:
> ** (com.github.artemanufrij.graphui:3): WARNING **: 20:33:13.482: GraphViz.vala:114: Failed to rename file “.5FPCWZ” to “”: g_rename() failed: No such file or directory

**Comment by nedrichards** [comment]
@bilelmoussaoui ping?

**Comment by bilelmoussaoui** [comment]
@nedrichards I have fixed the issue and it works pretty fine locally other than some small issues that are already reported upstream. This can finally be merged I guess.

bot, build com.github.artemanufrij.graphui

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.artemanufrij.graphui

**Pull request closed without merging**